CHANGELOG.md

```diff
@@ -28,7 +28,16 @@ All notable changes to this project will be documented in this file, in reverse
 
 ### Added
 
-- [#76](https://github.com/zendframework/zend-test/pull/76) adds support for PhpUnit 8
+- [#76](https://github.com/zendframework/zend-test/pull/76) adds support for PHPUnit 8
+  
+  When relying on PHPUnit installed as indirect dependency via zend-test, undesired
+  PHPUnit update to ^8.0 can happen on PHP ^7.2.  
+  Please declare direct dependency on `phpunit/phpunit` with suitable versions.
+  
+  If update causes errors with the message like "Fatal error: Declaration of *::setUp()
+  must be compatible with *::setUp(): void", `phpunit/phpunit`  can be declared as direct
+  dependency with the following command:
+  `composer require --dev phpunit/phpunit: "^7.5.12 || ^6.5.14 || ^5.7.14" --update-with-dependencies`
```

Comment:
In "normal" project dependency should be only on ONE phpunit version.

The same is in case `tearDown` method, I see it's just example, so not sure how specific we should be.

Also the command is wrong, when I run it I got:
> `Could not parse version constraint : Invalid version string "" `

I think it should be:
```console
$ composer require --dev phpunit/phpunit:"^7.5.12 || ^6.5.14 || ^5.7.14" --update-with-dependencies
```

(no space after `:`)

Reply:
Uh, yeah. that space was a typo
Well, i can't know which php version they use, so i specified the whole range for 5.6 and up